AniHyou/Screens/StaffDetails/ViewModels/Enums.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  Enums.swift
+//  AniHyou
+//
+//  Created by Noah Kovacs on 2/5/24.
+//
+
+import Foundation
+import SwiftUI
+
+public enum StaffOnList: String, Equatable, CaseIterable {
```

Comment:
I already had a picker for selecting three states, use it. [TriPicker](https://github.com/axiel7/AniHyou-iOS/blob/main/AniHyou/Views/TriPicker.swift)